src/controller/business.controller.ts

```diff
@@ -1,61 +1,61 @@
-import {Request, Response } from "express";
-import businessService from "../services/business.service";
-import { IRequest } from "../interface/IRequest.interface";
+import { Request, Response } from 'express';
+import businessService from '../services/business.service';
+import { IRequest } from '../interface/IRequest.interface';
 
-const {createBusiness, getBusinessData, updateBusiness} = businessService;
+const { createBusiness, getBusinessData, updateBusiness } = businessService;
 
 export default class businessController {
-  static async createNewBusiness (req: IRequest, res: Response): Promise<any> {
+  static async createNewBusiness(req: IRequest, res: Response): Promise<any> {
     try {
-      const {id} = req.decoded;
-      const userId = id
+      const { id } = req.decoded;
+      const userId = id;
       const data = req.body;
-      const newBusiness = await createBusiness(userId, data);
-      if(newBusiness) {
+      const newBusiness = await createBusiness(userId, data, req);
+      if (newBusiness) {
         return res.status(201).json({
-          message: "business created successfully!",
-          data: newBusiness
-        })
+          message: 'business created successfully!',
+          data: newBusiness,
+        });
       }
     } catch (error) {
       res.status(403).json({
```

Comment:
`return` is supposed to be here, right?
```suggestion
      return res.status(403).json({
```

Reply:
Yeah. I didn't make changes to that part of the code tho. I will add return